romancal/resample/tests/test_resample.py

```diff
@@ -712,6 +717,8 @@ def test_resampledata_do_drizzle_default_single_exposure_weight_array(
         many_to_one_model = output_models_many_to_one.borrow(0)
         assert np.any(many_to_one_model.weight > 0)
         assert np.any(many_to_many_model.weight > 0)
+        assert many_to_one_model.data[10, 40] == 0
```

Comment:
What is this line testing? The referenced pixel at 10, 40 (seen as the single highlighted pixel in the upper left) falls in the "fill" region of the output image.
<img width="556" alt="Screenshot 2025-02-03 at 1 47 59 PM" src="https://github.com/user-attachments/assets/b2e119a8-b234-46cb-bc18-606d2cd96037" />